src/Morpho.sol

```diff
@@ -377,10 +377,16 @@ contract Morpho is IMorpho {
         uint256 badDebtShares;
         if (position[id][borrower].collateral == 0) {
             badDebtShares = position[id][borrower].borrowShares;
-            uint256 badDebt = badDebtShares.toAssetsUp(market[id].totalBorrowAssets, market[id].totalBorrowShares);
-            market[id].totalSupplyAssets -= badDebt.toUint128();
-            market[id].totalBorrowAssets -= badDebt.toUint128();
-            market[id].totalBorrowShares -= badDebtShares.toUint128();
+            // Safe "unchecked" cast since badDebt <= totalBorrowAssets <= type(uint128).max
```

Comment:
```suggestion
            // Safe "unchecked" cast since the value is smaller than totalBorrowAssets
```
because badDebt is the result of the cast. But maybe it's clearer without this suggestion, even if it's a little incorrect

Reply:
I'm fine with both